src/routes/Raid.tsx

```diff
@@ -17,7 +17,7 @@ const userStatSorts: {
 }
 const userStatSortNames = Object.keys(userStatSorts)
 
-const ViewRaid = () => {
+const Raid = () => {
```

Comment:
Why the rename?

Reply:
Just wanted to perserve the Singular-Plural theme, I can change it back

Reply:
Cool, just want to make sure there aren't arbitrary "personal preference" changes or getting into "bikeshedding". 

I am always down for discourse and what not though.

Reply:
I was trying to void almost identical names - especially after the experience with HospitalRun... but if there are good reasons to avoid this, then I'm all ears!